File: ProblemCollection/src/kakao_internship2020/PressKeypad.js
function solution(numbers, hand) {
    let answer = '';
    let l = 10,
        r = 12;
    for (let i of numbers) {
        if (i == 0) i = 11;
        let countL = Math.abs(parseInt((l - i) / 3) + ((l - i) % 3));
        let countR = Math.abs(parseInt((r - i) / 3) + ((r - i) % 3));
        if (
            i % 3 == 1 ||
            (i % 3 == 2 &&
                (countL < countR || (countL == countR && hand == 'left')))
        ) {
            answer += 'L';
            l = i;
        } else {
            answer += 'R';
            r = i;
        }
    }
    return answer;
}

File: ProblemCollection/src/kakao_internship2020/PressKeypad.py
def solution(numbers, hand):
    answer = ''
    l = 10
    r = 12
    for i in numbers:
        if i == 0: i = 11
        countL = abs(int((l - i) / 3)) + abs(l - i) % 3
        countR = abs(int((r - i) / 3)) + abs(r - i) % 3
        if i % 3 == 1 or i % 3 == 2 and (countL < countR or countL == countR and hand == "left"):
            answer += 'L'
            l = i
        else:
            answer += 'R'
            r = i
    return answer
